webhooks.md

```diff
@@ -2,6 +2,139 @@
 
 When an action of particular type is performed in the system, a message is sent to the configured webhook, using HTTP POST and JSON body.
 
+## General message
+
+```javascript
+{
+    "EnterpriseId": "851df8c8-90f2-4c4a-8e01-a4fc46b25178",
+    "IntegrationId": "c8bee838-7fb1-4f4e-8fac-ac87008b2f90",
+    "Events": [
+        {
+            "Discriminator": "ServiceOrderUpdated",
+            "Value": {
+                "Id": "bfee2c44-1f84-4326-a862-5289598f6e2d"
+            }
+        },
+        {
+            "Discriminator": "IntegrationCreated",
+            "Value": {
+                "Id": "c8bee838-7fb1-4f4e-8fac-ac87008b2f90",
+                "AccessToken":"9E5E84E9974D4F169662AB2200F27CB1-00B343A0DDA725CACAC028E38E3EABF",
+                "Requestor": null
+            }
+        }
+    ],
+    "Entities": {
+        "ServiceOrders": [
+            {
+                "AdultCount": 2,
+                "AssignedResourceId": "20e00c32-d561-4008-8609-82d8aa525714",
+                "AssignedResourceLocked": false,
+                "BusinessSegmentId": null,
+                "CancelledUtc": null,
+                "ChannelNumber": "1337614414",
+                "ChannelManagerNumber": "01",
+                "ChannelManagerGroupNumber": "JX8PA2",
+                "ChannelManager": "AvailPro",
+                "ChildCount": 0,
+                "CompanionIds": [
```

Comment:
This forces us to load companionships which I don't think we should do. I wouldn't introduce this to webhooks. Cannot we deprecate this property?